Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -105,14 +112,24 @@ final class RewardCell: UICollectionViewCell, ValueCell {
         transform = .identity
       }
 
-      _ = self
+      _ = self.rewardCardContainerView
         |> \.transform .~ transform
     }
 
     animator.startAnimation()
   }
 }
 
+extension RewardCell: UIScrollViewDelegate {
+  func scrollViewDidScroll(_ scrollView: UIScrollView) {
+    let cardContainerViewY = self.rewardCardContainerView.frame.origin.y
+    let yOffset = scrollView.contentOffset.y - scrollViewTopInset - cardContainerViewY
```

Comment:
Are you able to get the `scrollViewTopInset` dynamically from ` self.scrollView.contentInset.top`?